worker-markup-core/src/test/resources/xml/EmailUntagHavingDividerWithEmptyHeadersEmptyBody.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2018 Micro Focus or one of its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<root><ACCOUNTS>{"handles":["\"dan j hyvl\" &lt;dan j hyvl&gt;"],"identifier":"\"dan j hyvl\" &lt;dan j hyvl&gt;","sender":true,"identifier_type":"EML","display_name":"\"Dan J Hyvl\" &lt;Dan J Hyvl&gt;"}</ACCOUNTS><ACCOUNTS>{"handles":["CWPfister@cps-satx.com"],"identifier":"CWPfister@cps-satx.com","sender":false,"identifier_type":"EML","display_name":"Pfister Christian W."}</ACCOUNTS><ADDRESS_DISPLAY_FROM>"Dan J Hyvl" &lt;Dan J Hyvl&gt;</ADDRESS_DISPLAY_FROM><ADDRESS_DISPLAY_TO>Pfister Christian W.</ADDRESS_DISPLAY_TO><ADDRESS_FROM>dan j hyvl &lt;dan j hyvl&gt;</ADDRESS_FROM><ADDRESS_TO>cwpfister@cps-satx.com</ADDRESS_TO><ARCHIVE_ID>50fcf7df-3f9c-4ebc-86fb-c09669a01158/cbf14d0c-ac87-45c7-8d47-a36d6cbb8e00.caf</ARCHIVE_ID><BATCH_ID>71</BATCH_ID><BINARY_HASH_SHA1>464c0fb8695eefb048e59d2b57673bffef757c2e</BINARY_HASH_SHA1><CAF_INDEXING_TOKEN>{"INDEX_ROUTE":"1022","INDEX_NAME":"morvinnew_item_1"}</CAF_INDEXING_TOKEN><CAF_MAIL_CONVERSATION_INDEX>AcnSAfKIY9ehzkbQT9eazV9TYssN5A==</CAF_MAIL_CONVERSATION_INDEX><CAF_MAIL_CONVERSATION_INDEX_PARSED>{"headerDate":"2009-05-11T06:30:16.256Z","guid":"63d7a1ce-46d0-4fd7-9acd-5f5362cb0de4","childMessages":[]}</CAF_MAIL_CONVERSATION_INDEX_PARSED><CAF_WORKFLOW_ACTION>1</CAF_WORKFLOW_ACTION><CAF_WORKFLOW_ID>1</CAF_WORKFLOW_ID><CAF_WORKFLOW_OUTPUT_PARTIAL_REFERENCE>50fcf7df-3f9c-4ebc-86fb-c09669a01158</CAF_WORKFLOW_OUTPUT_PARTIAL_REFERENCE><CAF_WORKFLOW_PROJECT_ID>1</CAF_WORKFLOW_PROJECT_ID><CAF_WORKFLOW_TENANT_ID>morvinnew</CAF_WORKFLOW_TENANT_ID><COLLECTION_STATUS>COLLECTED</COLLECTION_STATUS><CONTENT><email>Yes
```

Comment:
The rest of the testcase files only have the `<CONTENT>` element and that is all that you require as well.  Drop all the redundant elements.

Reply:
This comment still hasn't been addressed.